test/e2e/framework/util.go

```diff
@@ -353,17 +353,17 @@ func SkipUnlessTaintBasedEvictionsEnabled() {
 
 // SkipIfContainerRuntimeIs skips if the container runtime is included in the runtimes.
 func SkipIfContainerRuntimeIs(runtimes ...string) {
-	for _, runtime := range runtimes {
-		if runtime == TestContext.ContainerRuntime {
-			skipInternalf(1, "Not supported under container runtime %s", runtime)
+	for _, containerruntime := range runtimes {
```

Comment:
```suggestion
	for _, containerRuntime := range runtimes {
```

or just
```suggestion
	for _, rt := range runtimes {
```

go encourages short names for local variables with limited scope